test/functional/test_runner.py

```diff
@@ -401,8 +401,9 @@ def main():
         for test in tests:
             script = test.split("/")[-1]
             script = script + ".py" if ".py" not in script else script
-            if script in ALL_SCRIPTS:
-                test_list.append(script)
+            matching_scripts = [s for s in ALL_SCRIPTS if s.startswith(script)]
+            if matching_scripts:
+                test_list.extend(matching_scripts)
```

Comment:
```suggestion
            test_list.extend([s for s in ALL_SCRIPTS if s.startswith(script)])
```

Reply:
That change prevents the error that no test matches though?

Reply:
Right. Sorry, missed that there is an `else`.